app/views/SearchMessagesView/index.tsx

```diff
@@ -254,16 +256,15 @@ class SearchMessagesView extends React.Component<ISearchMessagesViewProps, ISear
 		);
 	};
 
-	renderItem = ({ item }: { item: IMessage }) => {
+	renderItem = ({ item }: { item: IMessageFromServer | TMessageModel }) => {
+		const message = item as TMessageModel;
 		const { user, baseUrl, theme, useRealName } = this.props;
 		return (
 			<Message
-				// @ts-ignore IMessage | TMessageModel?
-				item={item}
+				item={message}
 				baseUrl={baseUrl}
 				user={user}
 				timeFormat='MMM Do YYYY, h:mm:ss a'
-				isHeader
```

Comment:
Why did you delete this property?

Reply:
the property isHeader doesn't exist anymore! I searched where this was modified and was modified when Diego added a get isHeader in `app/containers/message/index.tsx`. What do you think?

https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/880/files#diff-2655a2c9c415ee7ce1aa809d9e64ff6bda856982c95a6ae7bb17dce5e4853135R123